pycqed/instrument_drivers/meta_instrument/heterodyne.py

```diff
@@ -465,6 +490,8 @@ def __init__(self, name, LO, AWG, acquisition_instr='CBox',
         self.mod_amp(.5)
         self._acquisition_delay = 0
         self.acquisition_delay(200e-9)
+        self._I_channel = 'ch3'
```

Comment:
This can be taken care of by specifying an `initial_value` if the `parameter_class=ManualParameter`